joomla/joomla-cms
# fix also the non sql install files menu entrys

Pull Request for Issue #13894 for the other two databases

### Summary of Changes

This syncs the backend main menu's over the testing data

### Testing Instructions

- Install all testing data via this package https://github.com/zero-24/joomla-cms/archive/fix_nonmysql_install_menu.zip
- make sure the backend menu's is complete

### Documentation Changes Required

None

@waader @alikon can you review / test this changes?
